playground/src/specs/streetlights.ts

```diff
@@ -140,7 +146,6 @@ channels:
       streetlightId:
         $ref: '#/components/parameters/streetlightId'
     publish:
-      operationId: turnOff
```

Comment:
Why remove the operationId? 🤔

Reply:
To check if without `operationId` everything is ok :)